config/routes.rb

```diff
@@ -56,6 +56,7 @@
       get '/edit' => 'application_form#edit', as: :application_edit
       get '/review' => 'application_form#review', as: :application_review
       get '/review/submitted' => 'application_form#review_submitted', as: :application_review_submitted
+      get '/review/submitted/:id' => 'application_form#review_previous_application', as: :application_review_previous
```

Comment:
What does the `:id` do here?

Reply:
I think we will need this for showing previous applications when there are more than 1. My current implementation doesn't reflect this. So if they have 3 previous applications it would pass the id specific to the first/second/third application.